language-server/src/test/java/org/sonarlint/languageserver/SonarLintTelemetryTest.java

```diff
@@ -150,17 +150,33 @@ public void upload_should_not_trigger_upload_when_disabled() {
   }
 
   @Test
-  public void usedAnalysis_should_trigger_usedAnalysis_when_enabled() {
+  public void analysisDoneOnMultipleFiles_should_trigger_analysisDone_when_enabled() {
     when(engine.isEnabled()).thenReturn(true);
-    telemetry.usedAnalysis();
+    telemetry.analysisDoneOnMultipleFiles();
     verify(engine).isEnabled();
-    verify(engine).usedAnalysis();
+    verify(engine).analysisDoneOnMultipleFiles();
   }
 
   @Test
-  public void usedAnalysis_should_not_trigger_usedAnalysis_when_disabled() {
+  public void analysisDoneOnMultipleFiles_should_not_trigger_usedAnalysis_when_disabled() {
```

Comment:
Probably `s/usedAnalysis/analysisDoneOnMultipleFiles/`. In the other similar test case names too.